prompb/io/prometheus/client/metrics.proto

```diff
@@ -51,6 +51,8 @@ message Gauge {
 message Counter {
   double   value    = 1;
   Exemplar exemplar = 2;
+
+  google.protobuf.Timestamp  created_timestamp = 3;
```

Comment:
Should it go on `Metric` rather than on the individual sub-components?
And be an `int64` like `timestamp_ms`?

Reply:
I'm basically copying what was already merged in [client_model](https://github.com/prometheus/client_model/pull/66). Unfortunately the proto is duplicated in this repo 😕

Reply:
Yeah, this is indeed just a port to proto3 from client_model. (And FTR, since created_timestamp only exists for _some_ metric types, it would be a mistake to put in on Metric.)